src/main/java/pl/betoncraft/betonquest/compatibility/citizens/CitizensParticle.java

```diff
@@ -75,22 +82,81 @@ public void run() {
 		enabled = true;
 	}
 
+
 	@Override
 	public void run() {
-		for (NPC npc : npcs) {
-			if (npc == null) {
-				continue;
-			}
-			Entity e = npc.getEntity();
-			if (e == null) {
-				continue;
-			}
-			Location loc = e.getLocation().clone();
-			loc.setPitch(-90);
-			Compatibility.getEffectManager().start(name, section, loc);
-		}
+	    // Bukkit.getLogger().info("Particle: RUN:89: Online Players: " + Bukkit.getOnlinePlayers().size() + "NPC:" + npcs.size());
+        for (NPC npc : npcs) {
+            if (npc == null) {
+                continue;
+            }
+            Entity e = npc.getEntity();
+            if (e == null) {
+                continue;
+            }
+            // Bukkit.getLogger().info("NPC Name: " + npc.getName());
+            Location loc = e.getLocation().clone();
+            loc.setPitch(-90);
+            // Compatibility.getEffectManager().start(name, section, loc);
```

Comment:
If that's commented out then this whole loop is useless. Besides, you can't just remove the default behavior, it should still work the same way it was when people upgrade the plugin.

I would suggest displaying both animations at the same time - if someone wants to just use conditioned effects he can simply disable the main, global effect in his config.